datalake-commons/src/main/scala/bio/ferlab/datalake/commons/config/ConfigurationWriter.scala

```diff
@@ -25,15 +25,19 @@ object ConfigurationWriter {
 
     val content = toHocon(conf)
 
+    val contentWithEnvVariable = content
```

Comment:
I'm not 100% happy with this solution and I would prefer a regex replacement with group like : 
replace "${?...}" by ${?...} but didn't succeed in doing that.
I have the regex (`^\"\$\{\??[A-Za-z_]*\}\"$`) but i'm not able to use it with groups, open to discuss about that